compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -369,6 +369,8 @@ void addGPUTileAndFusePassPipeline(OpPassManager &funcPassManager,
   tileAndDistributeToWorkgroup(funcPassManager, /*useForall=*/true,
                                /*convertToDpsOptions=*/std::nullopt);
 
+  funcPassManager.addPass(createLoweringConfigInterpreterPass());
```

Comment:
Is this where we want to run this pass?

Reply:
Yes this is the right place. I can add a comment explaining why.